pom.xml

```diff
@@ -95,13 +95,18 @@ under the License.
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<hadoop.version>2.8.5</hadoop.version>
+		<flink.XmxITCase>2048m</flink.XmxITCase>
+		<flink.XmxUnitTest>1024m</flink.XmxUnitTest>
 		<!-- Need to use a user property here because the surefire
 			 forkCount is not exposed as a property. With this we can set
 			 it on the "mvn" commandline in travis. -->
-		<flink.forkCount>1C</flink.forkCount>
+		<!-- Number of forkCounts for ITCase and UnitTest should take into account allocated memory
+			 to the jvm (-Xmx) and the available memory on the machine running the test -->
+		<flink.forkCountITCase>1C</flink.forkCountITCase>
+		<flink.forkCountUnitTest>2C</flink.forkCountUnitTest>
 		<!-- Allow overriding the fork behaviour for the expensive tests in flink-tests
 			 to avoid process kills due to container limits on TravisCI -->
-		<flink.forkCountTestPackage>${flink.forkCount}</flink.forkCountTestPackage>
+		<flink.forkCountTestPackage>${flink.forkCountITCase}</flink.forkCountTestPackage>
```

Comment:
we should get rid of `forkCountTestPackage`; it has been identical to the usual forkCount for as long as I can remember.